hack/verify-staging-imports.sh

```diff
@@ -79,7 +79,7 @@ if grep -rq '// import "k8s.io/kubernetes/' 'staging/'; then
 	exit 1
 fi
 
-for EXAMPLE in vendor/k8s.io/client-go/examples/{in-cluster,out-of-cluster,third-party-resources}; do
+for EXAMPLE in vendor/k8s.io/client-go/examples/{authenticate-in-cluster,authenticate-out-of-cluster,third-party-resources}; do
```

Comment:
The server url and port is also set up differently, so it's not just authenticate. Maybe call them in/out-cluster-client-configuration?